README.md

```diff
@@ -174,6 +174,10 @@ VARIABLE_WITH_DEFAULT: !var:default='defaultvalue' path/to/variable
     summon -D ENV=production --yaml 'SQL_PASSWORD: !var env/$ENV/db-password' deploy.sh
     ```
 
+* `--yaml` secrets.yml as a literal string.
+
+    This flag is used to put the secrets.yml variables on the command line (see example above).
```

Comment:
Codeclimate is complaining about line length here

Reply:
updated